### PR TITLE
add googledrivedownloader, pytorch_geometric

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/torch-geometric/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 131359fb1539ce5c837e8eb2dce8f7abd75294ae0ce44963648cd8646cfde6e1
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```
Comment (chrisburr) [inline recipes/torch-geometric/meta.yaml:15]: It's okay if Windows CI fails for noarch packages with missing dependencies
```suggestion
  noarch: python
```

Comment (rluria14) [reply recipes/torch-geometric/meta.yaml]: thanks, just updated

**Comment by rluria14** [comment]
@conda-forge/staged-recipes this recipe is ready for a final review

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**